code/homework3.txt

```diff
@@ -0,0 +1,26 @@
+for (var i=0;i<=20;i++)
+{
+var result=i*9;
+console.log(i+"*"+"9"+"="+result);
```

Comment:
Cool! You can also try something like `console.log(i + " * 9 = " + result);` and not have to concat multiple strings since `" * 9 = "` is not separated by any variables.